lib/logger.js

```diff
@@ -222,22 +222,22 @@ Logger.prototype._bufferLog = function(message) {
 
     if (!this._flusher) {
         debug('No scheduled flush. Scheduling for %d ms from now.', configs.FLUSH_INTERVAL);
-        this._flusher = setTimeout(bind(this._flush, this, (err) => {
-            if (err) {
-                debug('Received an error while flushing...' + err);
-            }
-        }), this._flushInterval);
+        this._flusher = setTimeout(() => {
+            this._flush();
+        }, this._flushInterval);
     }
 };
 
-Logger.prototype._flush = function(cb) {
+Logger.prototype._flush = function(callback) {
+    if (!callback || typeof callback !== 'function') {
+        callback = debug;
```

Comment:
I think this is the right way of handling `no callback` situation - `_flush` is internal method and not exposed to be used by clients - so, first of all, we shouldn't expect `no callback` situation - but if we expect, it would be better to just throw an error.